x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -116,6 +118,11 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 
 	workersRegrets := make([]alloraMath.Dec, 0)
 
+	topic, err := args.K.GetTopic(args.Ctx, args.TopicId)
+	if err != nil {
+		return errorsmod.Wrapf(err, "failed to get topic")
+	}
+
```

Comment:
You already have the `AlphaRegret` in the args; you don't need to fetch the topic again.

Reply:
good point!